src/Views/Components/AdminUI/api.ts

```diff
@@ -32,4 +32,39 @@ const usePostRequest = (endpoint: string, apiNonce: string, successMessage: stri
 
     return {postData, result};
 };
-export {usePostRequest};
+
+const useGetRequest = (endpoint: string, apiNonce: string, successMessage: string, errorMessage: string) => {
```

Comment:
We are currently passing empty strings for the success & error messages. Im not sure we plan on using them to display anything on the frontend but I may be wrong. If not we probably can just remove it.